pages/search.tsx

```diff
@@ -66,101 +69,155 @@ export default function Search({
-                      fadeIn: data.found,
-                      hidden: !data.found,
-                    })}
-                  >
-                    Showing {data.products.length} results{' '}
-                    {q && (
-                      <>
-                        for "<strong>{q}</strong>"
-                      </>
-                    )}
-                  </span>
-                  <span
-                    className={cn('animated', {
-                      fadeIn: !data.found,
-                      hidden: data.found,
-                    })}
-                  >
-                    {q ? (
-                      <>
-                        There are no products that match "<strong>{q}</strong>"
-                      </>
-                    ) : (
-                      <>
-                        There are no products that match the selected category &
-                        designer
-                      </>
-                    )}
-                  </span>
-                </>
-              ) : q ? (
-                <>
-                  Searching for: "<strong>{q}</strong>"
-                </>
-              ) : (
-                <>Searching...</>
+            <div className="mb-6 transition ease-in duration-75">
+              <div className="flex justify-between">
+                {data ? (
+                  <>
+                    <span
+                      className={cn('animated', {
+                        fadeIn: data.found,
+                        hidden: !data.found,
+                      })}
+                    >
+                      Showing {data.products.length} results{' '}
+                      {q && (
+                        <>
+                          for "<strong>{q}</strong>"
+                        </>
+                      )}
+                    </span>
+                    <span
+                      className={cn('animated', {
+                        fadeIn: !data.found,
+                        hidden: data.found,
+                      })}
+                    >
+                      {q ? (
+                        <>
+                          There are no products that match "<strong>{q}</strong>"
+                        </>
+                      ) : (
+                        <>
+                          There are no products that match the selected category &
+                          designer
+                        </>
+                      )}
+                    </span>
+                  </>
+                ) : q ? (
+                  <div>
+                    Searching for: "<strong>{q}</strong>"
+                  </div>
+                ) : (
+                  <>Searching...</>
+                )}
+
+                <button
+                  className="md:invisible text-accents-8 outline-none"
+                  onClick={() => setShowMobileMenu(prev => !prev)}
+                >
+                  Filter & Sort
```

Comment:
I think the better choose for button is an filter icon.

Reply:
Me too, I didn't apply on first hand because I was running out of time. Now it's an icon. Updated PR description image too.